### PR TITLE
Version 5.2

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### Reference Issues/PRs
#95 #149 #151
#### What does this implement/fix? Explain your changes.
Version 5.2

### CODE REVIEW

**Comment by sepandhaghighi** [comment]
@sadrasabouri Hi
Please complete this review until **Wednesday  - 2021-05-05 - 11:59 AM GMT+3:30**
**Notice:** Only review, don't need to merge

File: README.md [hunk 1]
```diff
@@ -829,14 +829,31 @@ or send an email to [info@4r7.ir](mailto:info@4r7.ir "info@4r7.ir").
...
+<h4>Ripple</h4>
+`rN7ZuRG7HDGHR5nof8nu5LrsbmSB61V1qq`
+<h4>Binance Coin</h4>
+`bnb1zglwcf0ac3d0s2f6ck5kgwvcru4tlctt4p5qef`
+<h4>Tether</h4>
+`0xcD4Db18B6664A9662123D4307B074aE968535388`
+<h4>Dash</h4>
+`Xd3Yn2qZJ7VE8nbKw2fS98aLxR5M6WUU3s`
+<h4>Stellar</h4>
+`GALPOLPISRHIYHLQER2TLJRGUSZH52RYDK6C3HIU4PSMNAV65Q36EGNL`
```
Comment (sadrasabouri) [inline README.md]: Nice job.
Can't we make some link kind of link which direct who wants to donate to a special website which he/she can donate us much easier than copy-paste the wallet?

**Review by sadrasabouri — APPROVED** [review]
Thank you, ready to go :rocket: 
I left a tiny comment. Hope it can help us.

File: README.md (hunk 1 above)
Comment (sepandhaghighi) [reply README.md]: We only accept cryptocurrency, so there is no problem ;-)

**Pull request merged**